simulation/common/helpers.py

```diff
@@ -389,11 +384,11 @@ def compute_elevation_angle_math(declination_angle, hour_angle, latitude):
     Returns: The elevation angle in degrees
     """
     term_1 = np.sin(np.radians(declination_angle)) * \
-        np.sin(np.radians(latitude))
+             np.sin(np.radians(latitude))
 
     term_2 = np.cos(np.radians(declination_angle)) * \
-        np.cos(np.radians(latitude)) * \
-        np.cos(np.radians(hour_angle))
+             np.cos(np.radians(latitude)) * \
+             np.cos(np.radians(hour_angle))
 
```

Comment:
I agree that if the lines are split into two we should add tabs. But these lines are not really too long, I think we can just make them one line.